$NetBSD: patch-ae,v 1.22 2009/02/11 10:21:33 markd Exp $

&data issue filed as http://www.cups.org/str.php?L3079 and fixed in
cups svn 28 minutes later.

--- cups/http.c.orig	2008-08-01 14:33:16.000000000 -0400
+++ cups/http.c
@@ -1175,7 +1175,7 @@ httpInitialize(void)
   for (i = 0; i < sizeof(data); i ++)
     data[i] = rand(); /* Yes, this is a poor source of random data... */
 
-  RAND_seed(&data, sizeof(data));
+  RAND_seed(data, sizeof(data));
 #endif /* HAVE_LIBSSL */
 }
 
